src/ensembl/src/content/app/entity-viewer/state/bookmarks/entityViewerBookmarksSelectors.ts

```diff
@@ -0,0 +1,9 @@
+import { RootState } from 'src/store';
+
+export const getAllPreviouslyViewedEntities = (state: RootState) => {
+  return state.entityViewer.bookmarks.previouslyViewed;
+};
+
+export const getPreviouslyViewedEntities = (state: RootState, genomeId: string) => {
+  return getAllPreviouslyViewedEntities(state)[genomeId] || [];
+};
```

Comment:
We don't need to use `return` or curly braces for single line arrow functions e.g:

```export const getAllPreviouslyViewedEntities = (state: RootState) => state.entityViewer.bookmarks.previouslyViewed;```